### PR TITLE
Fizzbuzz with if, for complete

### PR DESCRIPTION
**Pull request by ulgoon**
please merge my pull request.

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,100+1):
+    if i%15==0:
+        print('fizzbuzz')
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:3]: Capitalize f to F

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,100+1):
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:1]: please leave commentary

Comment (ulgoon) [reply fizzbuzz.py:1]: Oh got it..

File: fizzbuzz.py (hunk 1 above)
Comment (ulgoon) [reply fizzbuzz.py:3]: Okaaaaaaay.

**Pull request merged**